java/src/jmri/jmrit/display/CoordinateEdit.java

```diff
@@ -332,7 +332,7 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
     public void initSetLevel() {
         oldX = pl.getDisplayLevel();
         textX = new javax.swing.JLabel();
-        textX.setText("level= " + pl.getDisplayLevel());
+        textX.setText(Bundle.getMessage("Level") + " = " + pl.getDisplayLevel());
```

Comment:
> Thanks @rhwood <https://github.com/rhwood> , interesting read. Is there any place in JMRI where we already apply MassageFormat?
> 
jmri.jmrix.loconet.duplexgroup.DuplexGroup.properties is the only .properties file I can find that is using a MessageFormat formatter.

(I used the command ```find . -name \*.properties -exec grep -H '{[0-9],' {} \;``` to find it)